pkg/cmd/openshift-sdn/cmd.go

```diff
@@ -0,0 +1,269 @@
+package openshift_sdn
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	"k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig"
+	"k8s.io/kubernetes/pkg/util/interrupt"
+
+	"github.com/openshift/library-go/pkg/serviceability"
+	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
+	_ "github.com/openshift/origin/pkg/cmd/server/apis/config/install"
+	configapilatest "github.com/openshift/origin/pkg/cmd/server/apis/config/latest"
+	networkvalidation "github.com/openshift/origin/pkg/cmd/server/apis/config/validation/network"
+	sdnnode "github.com/openshift/origin/pkg/network/node"
+	sdnproxy "github.com/openshift/origin/pkg/network/proxy"
+	"github.com/openshift/origin/pkg/version"
+)
+
+// OpenShiftSDN stores the variables needed to initialize the real networking
+// processess from the command line.
+type OpenShiftSDN struct {
+	ConfigFilePath            string
+	KubeConfigFilePath        string
+	URLOnlyKubeConfigFilePath string
+
+	NodeConfig  *configapi.NodeConfig
+	ProxyConfig *kubeproxyconfig.KubeProxyConfiguration
+
+	informers *informers
+	OsdnNode  *sdnnode.OsdnNode
+	OsdnProxy *sdnproxy.OsdnProxy
+}
+
+var networkLong = templates.LongDesc(`
+	Start OpenShiftSDN node components. This includes the service proxy.
+
+	This will also read the node name from the environment variable K8S_NODE_NAME.`)
+
+func NewOpenShiftSDNCommand(errout io.Writer) *cobra.Command {
+	sdn := &OpenShiftSDN{}
+
+	cmd := &cobra.Command{
+		Use:   "network",
+		Short: "Start OpenShiftSDN",
```

Comment:
probably change `Use` too? (And maybe a space in "OpenShift SDN".) And btw note that the test-cmd failure wasn't a flake; it's checking the output of "openshift-cmd --help", so that test needs to be updated to match whatever you put here.